packages/apps/human-app/frontend/src/shared/utils/time.ts

```diff
@@ -0,0 +1,22 @@
+export type ProposalStatus = 'pending' | 'active';
+
+export function getProposalStatus(
```

Comment:
It's better to have this util in some other place because it doesn't have anything in common with `time`.

Reply:
@flopez7 actually it could be simple to just add proper status in `fetchProposal`, like:
```
const activeProposals = list.filter(
  (p) => p.voteStart <= now && now < p.voteEnd
);
if (activeProposals.length > 0) {
  const earliestEndingProposal = activeProposals.sort((a, b) => a.voteEnd - b.voteEnd)[0];
  return {
    ...earliestEndingProposal,
    status: 'active',
  };
}

const pendingProposals = list.filter((p) => now < p.voteStart);
if (pendingProposals.length > 0) {
  const earliestUpcomingProposal = activeProposals.sort((a, b) => a.voteEnd - b.voteEnd)[0];
  return {
    ...earliestUpcomingProposal,
    status: 'pending',
  };
}

return null;
```
So you don't even need this util